src/test/java/com/wikia/webdriver/common/dataprovider/VisualEditorDataProvider.java

```diff
@@ -92,6 +92,11 @@ public By getTag() {
     ADVANCED
   }
 
+  public enum ImageSize {
```

Comment:
Enums are lists of constants. When you need a predefined list of values which do not represent some kind of numeric or textual data, you should use an enum.